src/routes/console/project-[project]/overview/platforms/wizard/android/step2.svelte

```diff
@@ -12,7 +12,23 @@
 <WizardStep>
     <svelte:fragment slot="title">Get the SDK</svelte:fragment>
     <p>First, add this to your root level build.gradle file:</p>
-    <Code label="Groovy" labelIcon="android" language="kotlin" code={example1} withCopy />
+    <div class="u-margin-block-start-16">
+        <Code
```

Comment:
Do all/most `Code` components require top margin? We could bake it directly into the component